### ISSUE TITLE
Support for Linkvertise (AdFly's replacement)

### ISSUE THREAD

**Issue by Goldninja100gn** [HeroCC/yourls-conditional-urlads#9]
With AdFly merging with Linkvertise, I am wondering if it is possible to get support with Linkvertise as well. Idk if AdFly will still work once an account is merged, but it would be useful since it seems they are making us all merge our accounts.

**Comment by HeroCC** [comment]
Good idea. I don't have my development environment anymore, so _eventually_ I'll bring it back up and make these changes, but no estimate for the moment. I'm also happy to accept PRs. Everything you'd need to implement this is in [here](https://publisher.linkvertise.com/cdn/linkvertise.js).

**Comment by Goldninja100gn** [comment]
I'll have to look into that, not sure I'll have time to actually make a PR but would be fun to try when I get free time :D

**Comment by 8MiYile** [comment]
I will release a test version to try out the usability and when my server verifies that the changes pass, release
Because this proposal is related to a survey I had started, but no one responded, I was browsing here today and saw someone mention it, so I updated it
This is my previous public survey, no one participated [Link](https://github.com/8Mi-Tech/yourls-conditional-urlads/issues/2)
In fact, there are some things that you can ask me, and if I respond and definitely say I will add them, I will do testing in our branch and give you a Pull Request

**Comment by 8MiYile** [comment]
@HeroCC  @Goldninja100gn 
I have submitted the PullRequest
You are invited to test it together, and if it passes, please merge it directly into the main branch
Test:  a/ f/ o/ l/
https://github.com/HeroCC/yourls-conditional-urlads/pull/10

https://8mi.ink/a/OA8W27
https://8mi.ink/f/OA8W27
https://8mi.ink/o/OA8W27
https://8mi.ink/l/OA8W27
These URL should all work fine

this is Update Note
https://github.com/8Mi-Tech/yourls-conditional-urlads/issues/1
